deny.toml

```diff
@@ -57,6 +57,9 @@ license-files = [
 [sources]
 unknown-registry = "deny"
 unknown-git = "deny"
+private = [
```

Comment:
I'd say it's correct for cargo-deny to flag these patched dependencies.

Reply:
I think this is leftover from when we still had to refer to the pr version of operator-rs..